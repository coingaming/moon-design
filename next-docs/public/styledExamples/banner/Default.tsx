import { Banner } from '@heathmont/moon-core';

const Example = () => (
  <Banner title="Title here" description="Lorem ipsum dolor..." />
);

export default Example;
