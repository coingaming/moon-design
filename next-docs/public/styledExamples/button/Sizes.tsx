import { Button } from '@heathmont/moon-core';

const Example = () => (
  <>
    <Button size="xsmall">xSmall</Button>
    <Button size="small">Small</Button>
    <Button>Medium is default</Button>
    <Button size="large">Large</Button>
    <Button size="xlarge">xLarge</Button>
  </>
);

export default Example;
