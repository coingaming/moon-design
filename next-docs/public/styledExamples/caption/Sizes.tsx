import { Caption } from '@heathmont/moon-core';

const Example = () => (
  <>
    <Caption>Default font-size is 10px</Caption>
    <Caption size={9}>Size is 9px</Caption>
  </>
);

export default Example;
