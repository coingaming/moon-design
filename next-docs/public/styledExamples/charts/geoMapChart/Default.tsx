// @ts-nocheck
import React from 'react';
import { Filter, GeoMapChart } from '@heathmont/moon-charts';
import { SingleItemSelect } from '@heathmont/moon-components';

const Example = () => {
  const data = [
    { label: 'Estonia', code: 'EST', value: 120 },
    { label: 'Russia', code: 'RUS', value: 100 },
    { label: 'Belarus', code: 'BLR', value: 80 },
    { label: 'Argentina', code: 'ARG', value: 60 },
    { label: 'USA', code: 'USA', value: 40 },
  ];

  const filter = (
    <Filter title="by GGR">
      {() => (
        <SingleItemSelect
          onChange={() => {}}
          value="ggr"
          options={[
            { label: 'Active players', value: 'actives' },
            { label: 'GGR', value: 'ggr' },
            { label: 'Bets', value: 'bets' },
          ]}
        />
      )}
    </Filter>
  );

  return (
    <GeoMapChart
      title="Top Countries"
      data={data}
      filter={filter}
      hasUpdates={true}
      onUpdate={() => {}}
      onShare={() => {}}
      onExpand={() => {}}
    />
  );
};

export default Example;
