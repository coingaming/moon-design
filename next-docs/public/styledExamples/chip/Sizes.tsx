import { Chip } from '@heathmont/moon-core';

const Example = () => (
  <>
    <Chip size="small">Small</Chip>
    <Chip>Medium is Default</Chip>
  </>
);

export default Example;
