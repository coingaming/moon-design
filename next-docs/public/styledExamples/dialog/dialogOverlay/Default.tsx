import React from 'react';
import { DialogOverlay } from '@heathmont/moon-components';
import { Button } from '@heathmont/moon-core';

const Example = () => {
  const [showDialog, setShowDialog] = React.useState(false);

  return (
    <>
      <Button variant="tertiary" onClick={() => setShowDialog(true)}>
        Open Dialog
      </Button>
      <DialogOverlay
        isOpen={showDialog}
        onDismiss={() => setShowDialog(false)}
      />
    </>
  );
};

export default Example;
