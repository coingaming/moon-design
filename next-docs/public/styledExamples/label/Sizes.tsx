import React from 'react';
import { Label } from '@heathmont/moon-core';

const Example = () => (
  <>
    <Label size="twoxsmall">Twoxsmall</Label>
    <Label>Xsmall</Label>
    <Label size="twoxsmall" isUppercase={false}>
      Twoxsmall
    </Label>
    <Label isUppercase={false}>Xsmall</Label>
  </>
);

export default Example;
