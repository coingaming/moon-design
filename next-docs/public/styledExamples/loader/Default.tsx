import React from 'react';
import { Loader } from '@heathmont/moon-core';

const Example = () => <Loader />;

export default Example;
