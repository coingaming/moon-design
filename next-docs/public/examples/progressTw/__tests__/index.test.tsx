/**
 * @jest-environment jsdom
 */

import React from 'react';
import renderer from 'react-test-renderer';
import Default from '../Default';
import Colors from '../Colors';
import Values from '../Values';
import Sizes from '../Sizes';
import {
  moonDesignDark,
  moonDesignLight,
  ThemeProvider,
} from '@heathmont/moon-themes';

const renderWithTheme = (component: JSX.Element) => (
  <ThemeProvider theme={moonDesignLight}>{component}</ThemeProvider>
);

const renderWithDarkTheme = (component: JSX.Element) => (
  <ThemeProvider theme={moonDesignDark}>{component}</ThemeProvider>
);

const renderWithRtl = (component: JSX.Element) => (
  <div dir="rtl">{component}</div>
);

describe('Progress', () => {
  it('renders Default in Light theme', () => {
    const testRenderer = renderer.create(renderWithTheme(<Default />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders Default in Dark theme', () => {
    const testRenderer = renderer.create(renderWithDarkTheme(<Default />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders Default in RTL', () => {
    const testRenderer = renderer.create(renderWithRtl(<Default />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders Sizes in Light theme', () => {
    const testRenderer = renderer.create(renderWithTheme(<Sizes />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders Sizes in Dark theme', () => {
    const testRenderer = renderer.create(renderWithDarkTheme(<Sizes />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders Sizes in RTL', () => {
    const testRenderer = renderer.create(renderWithRtl(<Sizes />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders Values in Light theme', () => {
    const testRenderer = renderer.create(renderWithTheme(<Values />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders Values in Dark theme', () => {
    const testRenderer = renderer.create(renderWithDarkTheme(<Values />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders Values in RTL', () => {
    const testRenderer = renderer.create(renderWithRtl(<Values />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders Colors in Light theme', () => {
    const testRenderer = renderer.create(renderWithTheme(<Colors />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders Colors in Dark theme', () => {
    const testRenderer = renderer.create(renderWithDarkTheme(<Colors />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders Colors in RTL', () => {
    const testRenderer = renderer.create(renderWithRtl(<Colors />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
