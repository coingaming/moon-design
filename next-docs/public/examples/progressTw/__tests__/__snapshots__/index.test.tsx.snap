// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioButton renders Colors in Dark theme 1`] = `
Array [
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks h-4"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
      style={
        Object {
          "width": "75%",
        }
      }
    />
  </div>,
  <div
    className="w-full relative rounded-full overflow-hidden bg-whis h-4"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-dodoria"
      style={
        Object {
          "width": "75%",
        }
      }
    />
  </div>,
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-4"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-roshi"
      style={
        Object {
          "width": "75%",
        }
      }
    />
  </div>,
]
`;

exports[`RadioButton renders Colors in Light theme 1`] = `
Array [
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks h-4"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
      style={
        Object {
          "width": "75%",
        }
      }
    />
  </div>,
  <div
    className="w-full relative rounded-full overflow-hidden bg-whis h-4"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-dodoria"
      style={
        Object {
          "width": "75%",
        }
      }
    />
  </div>,
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-4"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-roshi"
      style={
        Object {
          "width": "75%",
        }
      }
    />
  </div>,
]
`;

exports[`RadioButton renders Sizes in Dark theme 1`] = `
Array [
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-0.5"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
      style={
        Object {
          "width": "75%",
        }
      }
    />
  </div>,
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-1"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
      style={
        Object {
          "width": "75%",
        }
      }
    />
  </div>,
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-2"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
      style={
        Object {
          "width": "75%",
        }
      }
    />
  </div>,
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-3"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
      style={
        Object {
          "width": "75%",
        }
      }
    />
  </div>,
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-4"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
      style={
        Object {
          "width": "75%",
        }
      }
    />
  </div>,
]
`;

exports[`RadioButton renders Sizes in Light theme 1`] = `
Array [
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-0.5"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
      style={
        Object {
          "width": "75%",
        }
      }
    />
  </div>,
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-1"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
      style={
        Object {
          "width": "75%",
        }
      }
    />
  </div>,
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-2"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
      style={
        Object {
          "width": "75%",
        }
      }
    />
  </div>,
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-3"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
      style={
        Object {
          "width": "75%",
        }
      }
    />
  </div>,
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-4"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
      style={
        Object {
          "width": "75%",
        }
      }
    />
  </div>,
]
`;

exports[`RadioButton renders Values in Dark theme 1`] = `
Array [
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-4"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
      style={
        Object {
          "width": "25%",
        }
      }
    />
  </div>,
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-4"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
      style={
        Object {
          "width": "50%",
        }
      }
    />
  </div>,
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-4"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
      style={
        Object {
          "width": "75%",
        }
      }
    />
  </div>,
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-4"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
      style={
        Object {
          "width": "100%",
        }
      }
    />
  </div>,
]
`;

exports[`RadioButton renders Values in Light theme 1`] = `
Array [
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-4"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
      style={
        Object {
          "width": "25%",
        }
      }
    />
  </div>,
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-4"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
      style={
        Object {
          "width": "50%",
        }
      }
    />
  </div>,
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-4"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
      style={
        Object {
          "width": "75%",
        }
      }
    />
  </div>,
  <div
    className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-4"
  >
    <span
      className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
      style={
        Object {
          "width": "100%",
        }
      }
    />
  </div>,
]
`;

exports[`RadioButton renders default in Dark theme 1`] = `
<div
  className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-4"
>
  <span
    className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
    style={
      Object {
        "width": "0%",
      }
    }
  />
</div>
`;

exports[`RadioButton renders default in Light theme 1`] = `
<div
  className="w-full relative rounded-full overflow-hidden bg-trunks/[.24] h-4"
>
  <span
    className="block absolute h-full rounded-full ltr:left-0 rtl:right-0 transition-all bg-piccolo"
    style={
      Object {
        "width": "0%",
      }
    }
  />
</div>
`;
